.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Run Tests
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: precice/precice
+    strategy:
+      matrix:
+        julia-version: ['^1.6']
+        julia-arch: [x64, x86]
```

Comment:
Are we testing something related to Arch Linux installation here? What does the command `julia-arch` do here exactly?

Reply:
This is the architecture on which julia is running. The matrix was taken from the [julia-actions github](https://github.com/julia-actions/setup-julia#32-bit-julia) where this was included. I figured, more test cases don't hurt.